chains/evm/executor/executor.go

```diff
@@ -180,23 +180,61 @@ func (e *Executor) watchExecution(ctx context.Context, cancelExecution context.C
 	}
 }
 
-func (e *Executor) executeProposal(proposals []*chains.Proposal, signatureData *common.SignatureData) (*ethCommon.Hash, error) {
+func (e *Executor) proposalBatches(msgs []*message.Message) ([]*Batch, error) {
+	batches := make([]*Batch, 1)
+	currentBatch := &Batch{
+		proposals: make([]*chains.Proposal, 0),
+		gasLimit:  0,
+	}
+	batches[0] = currentBatch
+
+	for _, m := range msgs {
+		prop, err := e.mh.HandleMessage(m)
+		if err != nil {
+			return nil, err
+		}
+
+		evmProposal := chains.NewProposal(prop.Source, prop.Destination, prop.DepositNonce, prop.ResourceId, prop.Data, prop.Metadata)
+		isExecuted, err := e.bridge.IsProposalExecuted(evmProposal)
+		if err != nil {
+			return nil, err
+		}
+		if isExecuted {
+			log.Info().Msgf("Prop %p already executed", prop)
```

Comment:
I know it was like this before, but I don't see reason why not use full word in logs
```suggestion
			log.Info().Msgf("Proposal %p already executed", prop)
```